pkg/util/github/github.go

```diff
@@ -235,6 +248,28 @@ func DownloadOpenShiftReleaseBinary(binaryType OpenShiftBinaryType, osType minis
 	return nil
 }
 
+func listDirExcluding(dir string, excludeRegexp string) ([]string, error) {
```

Comment:
hmm, one can for sure do so. I am not sure whether this is part of best practices or idiomatic in Go. Personally I am not a big fan.